src/main/scala/org/deepdive/ddlog/DeepDiveLogSemanticChecker.scala

```diff
@@ -156,6 +157,46 @@ object DeepDiveLogSemanticChecker extends DeepDiveLogHandler {
     }
   }
 
+  def checkSupervisionLableType(s: ExtractionRule, expType: VariableType, SupVariable: VarPattern, b: BodyAtom) {
+    b.terms.zipWithIndex.foreach {
+      case (SupVariable, index) => {
+        val expColumnType = if (expType.isInstanceOf[BooleanType.type]) "boolean" else "text"
```

Comment:
I think it should be `"int"` and the example should be fixed.  The [ext_training.py in the original DeepDive example is responsible for the text --> int mapping](https://github.com/HazyResearch/deepdive/blob/master/examples/chunking/udf/ext_training.py#L29), which isn't very clear from the translated ddlog version alone.